newFish.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once 'php-files/functions.php'
+?>
+<!DOCTYPE html>
```

Comment:
Run through https://validator.w3.org/nu/#textarea

9 errors and a warning to fix